README.md

```diff
@@ -104,39 +67,6 @@ Paintera can then be executed with the `paintera` command:
 ```
 paintera [[JGO ARG...] [JVM ARG...] --] [ARG...]
 ```
-By default, `org.slf4j:slf4j-simple` is used as `slf4j` binding for logging. This can be replaced by setting the `PAINTERA_SLF4J_BINDING` environment variable:
```

Comment:
What is the currently preferred way to test Paintera during development with various log levels?

Reply:
This is not supported any longer. I think it was removed in a previous commit. I think @hanslovsky may have some more insight on whether or not we need to support this again going forward.